### ISSUE TITLE
API schema uses same nickname for different endpoints.

### ISSUE THREAD

**Issue by GrahamDumpleton** [openshift/origin#10562]
The current schemas for model and apis in:
- https://github.com/openshift/origin/tree/master/api/swagger-spec

declare various API endpoints for different purposes which have the same nickname.

This will cause problems for automatically generated interfaces as not every endpoint has a unique nickname.

As example, swagger-codegen generates methods:

```
list_namespaced_image_stream
/oapi/v1/imagestreams

list_namespaced_image_stream_0
/oapi/v1/namespaces/{namespace}/imagestreams
```

because two endpoints used nickname `listNamespacedImageStream`.

Because order in which apis are returned in schema may change over time, can't even be guaranteed that the name for the method name for the end point will be the same as which gets the `_0` suffix might change.

For some cases at least, It looks like this problem may not have existed in older versions, but does for current version of schemas.

Someone should run:

```
grep "nickname" oapi-v1.json | sort
grep "nickname" api-v1.json | sort
```

and look at cases where more than one and determine if this is intended or not.

**Comment by mfojtik** [comment]
@openshift/api-review PTAL

**Comment by smarterclayton** [comment]
This seems like a bug in swagger.  Check the latest PR to Openshift-docs - this may have been fixed in the last rebase.

**Comment by GrahamDumpleton** [comment]
I don't know which PR you are referring to. And what will that mean for at least getting corrected schemas added to this repo?

For now can probably go back and grab older schemas off 3.2 (preview environment), as think they don't have issue, but will see.

**Comment by smarterclayton** [comment]
Should be fixed in #10983

**Comment by stevekuznetsov** [comment]
Fix merged.
